app/src/main/java/com/example/una/RecyclerViewModels/HorizontalModel.java

```diff
@@ -0,0 +1,22 @@
+package com.example.una.RecyclerViewModels;
+
+public class HorizontalModel {
+    String name,
```

Comment:
Just treat them as if they are completely different things and declare with type
```
String name;
String description;
```